core/src/main/scala/com/clairvoyant/restonomer/core/authentication/RestonomerAuthentication.scala

```diff
@@ -54,6 +56,57 @@ case class BasicAuthentication(
 
 }
 
+case class JwtAuthentication(
+    subject: Option[String] = None,
+    secretKey: Option[String] = None,
+    algo: Option[String] = None
+) extends RestonomerAuthentication {
+
+  implicit val clock: Clock = Clock.systemUTC
+
+  override def validateCredentials(): Unit = {
+    if (subject.isEmpty || secretKey.isEmpty || algo.isEmpty)
```

Comment:
Instead of isEmpty, use isBlank as it also checks for whitespace characters.

Reply:
resolved